Gemfile.lock

```diff
@@ -454,7 +457,8 @@ GEM
       rspec-mocks (~> 3.7.0)
       rspec-support (~> 3.7.0)
     rspec-support (3.7.1)
-    rubocop (0.54.0)
+    rubocop (0.57.1)
+      jaro_winkler (~> 1.5.1)
```

Comment:
🤔I wonder why rubocop now depends on `jaro_winkler`. Maybe they added some sort of "Did you mean?" feature?

Reply:
lol, how did you even notice that!! 🥇 😝

Reply:
turns out Rubocop has been doing it for quite a long time https://github.com/rubocop-hq/rubocop/pull/1293